azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeAudioSelectionMode.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import com.azure.android.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * Defines values for CallCompositeAudioSelectionMode.
+ */
+public final class CallCompositeAudioSelectionMode extends ExpandableStringEnum<CallCompositeAudioSelectionMode> {
```

Comment:
Would recommend both AudioSink (speakers) and AudioSource (microphones) as separate enums, even if they share the same values in some places.

I.e. 
Phone Mic/Phone Loud Speaker
Phone Mic/Phone Headset Speaker
Bluetooth Mic/Bluetooth Headset Speaker

and potentially in the future support for more mode combinations, i.e. like teams.

USB Mic/BT A2DP Speaker
USB MIC/Phone Speaker

etc.